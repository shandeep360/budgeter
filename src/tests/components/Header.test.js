import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../components/HeaderComponent';

test('Should render Header correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Header />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});