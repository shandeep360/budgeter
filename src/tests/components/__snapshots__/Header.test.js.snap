// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header correctly 1`] = `
<header>
  <h1>
    Budgeter
  </h1>
  <Unknown
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Home
  </Unknown>
  <Unknown
    activeClassName="is-active"
    to="/create"
  >
    Create Expense
  </Unknown>
</header>
`;
